controllers/utils/handlers/enqueueformatchingupdate.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handlers
+
+import (
+	"context"
+	"strings"
+
+	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	crtHandler "sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+// EnqueueForMatchingUpdate enqueues Request containing the Name and Namespace of the resource(s) specified in the
+// Status of the ReleasePlans and ReleasePlanAdmissions that are the source of the Event. The source of the event
+// triggers reconciliation of the parent resource.
+type EnqueueForMatchingUpdate struct{}
+
+var _ crtHandler.EventHandler = &EnqueueForMatchingUpdate{}
+
+// Create implements EventHandler
```

Comment:
```suggestion
// Create implements EventHandler.
```
Same for every other function.

Reply:
fixed